apps/web/src/components/AccountCard/AccountBalance.tsx

```diff
@@ -0,0 +1,25 @@
+import { AddIcon, ArrowDownIcon } from "@chakra-ui/icons";
+import { Box, Button, Flex, IconButton, Text } from "@chakra-ui/react";
+
+export const AccountBalance = () => (
+  <Box>
+    <Box marginBottom="4">
+      <Text marginBottom="1" color="gray.500" fontSize="sm">
```

Comment:
the problem with those string color literals is that they aren't type-checked. it's too easy to make a mistake and not notice it. like gray vs grey, gray.500 vs gray.5000.
also, in other places we use `mode(light.grey[50], dark.grey[50])(colorMode)` which IIRC does exactly the same

so, I'd suggest to stick to the 2nd option OR we could introduce our own hook like:

```typescript
import { useColorMode } from "@chakra-ui/react";
import { mode } from "@chakra-ui/theme-tools";
import { useCallback } from "react";

const useColor = () => {
  const colorMode = useColorMode();

  return useCallback(
    (light: number, dark?: number) =>
      mode(light.grey[light], dark.grey[dark || light])(colorMode),
    [colorMode]
  );
};
```

and then use it like this:

```typescript
const color = useColor();

<Flex background={color(100)} color={color(900, 800)}
```